apps/dot_ext/views/authorization.py

```diff
@@ -15,6 +17,31 @@ class AuthorizationView(DotAuthorizationView):
     login_url = "/mymedicare/login"
     template_name = "design_system/authorize.html"
 
+    def form_valid(self, form):
```

Comment:
Per Gage asking if there was a "cleaner" way to do this. I'm not seeing a nice way to make that cleaner. It would be nice if you could call the inherited (super) method from within form_valid() and do some post work/filtering for just the new items, but there's that redirect() on the return in the original. I think this looks good!